src/frontends/pytorch/src/op/cat.cpp

```diff
@@ -146,9 +146,16 @@ OutputVector translate_stack_fx(const NodeContext& context) {
         num_elements -= 1;
     }
 
+    OutputVector stack_inputs;
     for (size_t i = 0; i < num_elements; i++) {
-        auto stack_input =
-            context.mark_node(std::make_shared<v0::Unsqueeze>(context.get_input(static_cast<int>(i)), dim));
+        stack_inputs.push_back(context.get_input(static_cast<int>(i)));
+    }
+
+    if (const auto& u4_const = u4_compression_stack(stack_inputs, axis))
```

Comment:
Please add small comment why we do this here.

Reply:
done